VRDR/DeathRecord_ctors.cs

```diff
@@ -0,0 +1,665 @@
+using System;
```

Comment:
Is this file for constructors? I wasn't sure based on filename

Reply:
Is there a letter shortage? A good coding practice to adopt is to spell things out. The trade off of typing a few extra characters is well worth minimizing years of potential confusion on the part of people trying to understand.

Reply:
Does changing DeathRecord_codedProperties.cs to just DeathRecord_properties.cs work? What does the "coded" signify?